build.sbt

```diff
@@ -208,7 +208,6 @@ lazy val http2Support = project("akka-http2-support")
     )
   }
   .enablePlugins(JavaAgent, BootstrapGenjavadoc)
-  .disablePlugins(MimaPlugin) // experimental module still
```

Comment:
A bit unexpected to have this in the same PR.

Reply:
The relationship is that this would trigger #3135, so including it here proved that it fixes that problem.

I agree that now that we've verified that it's probably better to separate them again, rebased.